### DISCUSSION TITLE
Ventoy booting fist iso/img/efi automaticlly #824

### DISCUSSION THREAD

**Discussion by Admin-A-LLC** [ventoy/Ventoy#826] [Q&A]
I've noticed that on some of my computers, ventoy boots but then immediately boots the first file on the menu. Not sure how to collect debug files for this. As an attempt to circumvent it, I've created a rEFInd bootable ISO file and named it 1.iso so that it's at the top of the list. This gets me to a menu and even lets me fall back to the Ventoy boot menu, but none of the boot options work and it just reboot back into a cycle. Any way around this?

**Comment by steve6375** [comment]
Do you have a ventoy.json file on the drive? If so, do you set a timeout?

**Reply by Admin-A-LLC** [reply]
I didn't actually, but i'm experimenting with a ventoy.json file now. Do you have and example of the timeout in the json file? I'm not much of a programmer, just can read and guess.
